py/desimodel/inputs/throughput.py

```diff
@@ -18,7 +18,7 @@
 from . import docdb
 from ..io import datadir, findfile
 
-def update(testdir=None, desi347_version=13, desi334_version=3):
+def update(testdir=None, desi347_version=16, desi5501_version=3, desi5501_KOSI=True):
```

Comment:
Please update docstring to match these options.  In particular, what does desi_5501_KOSI=True or False mean?